src/Notepads.Controls/Notepads.Controls.csproj

```diff
@@ -11,7 +11,7 @@
     <AssemblyName>Notepads.Controls</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.19041.0</TargetPlatformVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.18362.0</TargetPlatformVersion>
```

Comment:
Is there any specific reason you are decreasing target version??

Reply:
It is because I don't have the latest SDK. Should I install it?

Reply:
Yes, install the latest sdk, Notepads depends on this to provide a some feature that aren't available in previous versions.

Reply:
Ok. I'll install it. Once I have done it, do I have to submit a new PR, right?

Reply:
You don't have to submit new pr, just revert the target platform changes you have made and push a new commit.

Reply:
Ok. I just finished to install it. Now, I will push a new commit. Thanks.